### PR TITLE
convert doc to md issue #450

### PR DESCRIPTION
**Pull request by sonaalKant**
Issue #450

### CODE REVIEW

File: docs/Contributor_scorecard.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Contributor - Scorecard
```
Comment (DanilYachmenev) [inline docs/Contributor_scorecard.md:1]: -> `# Contributor Scorecard`

File: docs/Contributor_scorecard.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Thank you for joining the Sorrentum project and contributing to it.
```
Comment (DanilYachmenev) [inline docs/Contributor_scorecard.md:3]: remove this line

File: docs/Contributor_scorecard.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  - Contributors that have a score less than 3 will be given a warning and if
+    they can't increase their score, they will be dropped from the project. Our
+    decision is final, everybody had a fair shot, but we can't have dragged on
+    discussions. In life, you have only one chance to give a first impression.
+    After you are dropped, you can take 6 months off to improve your skills and
+    then apply to contribute for the project again
+
+Thank you,
+
+The Sorrentum team
```
Comment (DanilYachmenev) [inline docs/Contributor_scorecard.md]: remove these ones as well

File: docs/Contributor_scorecard.md
Comment (DanilYachmenev) [inline docs/Contributor_scorecard.md:1]: lets name it with uppercased letters, looks good
`Contributor_Scorecard.md`

File: docs/Contributor_scorecard.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# Contributor - Scorecard
+
+Thank you for joining the Sorrentum project and contributing to it.
+
+We give feedback about every 2 weeks. The scores are published [<span
+class="underline">here</span>](https://docs.google.com/spreadsheets/d/1a8ypuO2ODOzjp9BaRN23HWa5P7ruTKs_gHZWaPYsvy4/edit#gid=2082207396)
```
Comment (DanilYachmenev) [inline docs/Contributor_scorecard.md:6]: Lets put the text under sections to improve readability a bit and to align with other docs
so here
```
## General
- We give feedback about every 2 weeks
- The scores are published [<span
class="underline">here</span>](https://docs.google.com/spreadsheets/d/1a8ypuO2ODOzjp9BaRN23HWa5P7ruTKs_gHZWaPYsvy4/edit#gid=2082207396)
```

File: docs/Contributor_scorecard.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  improvement will not only benefit your internship but also contribute to your
+  overall growth as a professional
+
+- Multiple people review your work and assess it as objectively as possible, so
+  please don't question your scores, but rather try to understand why you
+  receive the feedback and focus on how to improve the scores
+
+- We are a non-profit open source project so we have limited capacity and time.
+  We give an opportunity to contribute to everyone, but we can't support
+  contributors that don't contribute enough to the project
```
Comment (DanilYachmenev) [inline docs/Contributor_scorecard.md:27]:
```
## Reminders

- (4 main bullet points as they are)
```

File: docs/Contributor_scorecard.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+We give feedback about every 2 weeks. The scores are published [<span
+class="underline">here</span>](https://docs.google.com/spreadsheets/d/1a8ypuO2ODOzjp9BaRN23HWa5P7ruTKs_gHZWaPYsvy4/edit#gid=2082207396)
+
+Some reminders
+
+- The goal of this internship is to bridge the gap between college and the
+  workplace. We try to give you a preview of how your skills will be evaluated
+  and prepare you for being hired. As you have seen, we spend a lot of time
+  helping you improve your skills through code reviews, writing documentation,
+  suggesting resources to read, etc
```
Comment (DanilYachmenev) [inline docs/Contributor_scorecard.md:14]: -> `... etc.`

File: docs/Contributor_scorecard.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    with them.
+
+  - Contributors that have a score less than 3 will be given a warning and if
+    they can't increase their score, they will be dropped from the project. Our
+    decision is final, everybody had a fair shot, but we can't have dragged on
+    discussions. In life, you have only one chance to give a first impression.
+    After you are dropped, you can take 6 months off to improve your skills and
+    then apply to contribute for the project again
+
+The Sorrentum team
```
Comment (DanilYachmenev) [inline docs/Contributor_scorecard.md:44]: remove

File: docs/Contributor_scorecard.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+    lacking. Your mentor (indicated in the Google Sheet) can provide you
+    guidance for improving on Telegram. Feel free to have a short discussion
+    with them.
+
+  - Contributors that have a score less than 3 will be given a warning and if
+    they can't increase their score, they will be dropped from the project. Our
+    decision is final, everybody had a fair shot, but we can't have dragged on
+    discussions. In life, you have only one chance to give a first impression.
+    After you are dropped, you can take 6 months off to improve your skills and
+    then apply to contribute for the project again
```
Comment (DanilYachmenev) [inline docs/Contributor_scorecard.md:42]:
```
## Grades

- (bullet points bout grades)
```

File: docs/Contributor_scorecard.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+  We give an opportunity to contribute to everyone, but we can't support
+  contributors that don't contribute enough to the project
+
+  - A score of 4 or higher is considered contributing, indicating a good
+    performance
+
+  - If your score falls below 4, please focus on improving in areas you are
+    lacking. Your mentor (indicated in the Google Sheet) can provide you
+    guidance for improving on Telegram. Feel free to have a short discussion
+    with them.
```
Comment (DanilYachmenev) [inline docs/Contributor_scorecard.md:35]: no dot at the end of bullet point (`etc.` is exception))

File: docs/Contributor_scorecard.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
+# Contributor - Scorecard
+
```
Comment (DanilYachmenev) [inline docs/Contributor_scorecard.md:2]: after you reorg the text in sections, add a TOC following https://github.com/sorrentum/sorrentum/blob/master/docs/Documentation_about_guidelines.md#table-of-content-toc

**Review by DanilYachmenev — COMMENTED** [review]
nice, see the suggested formatting comments

File: docs/Contributor_Scorecard.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
+# Contributor Scorecard
+
+<!-- toc -->
```
Comment (sonaalKant) [inline docs/Contributor_Scorecard.md:3]: Is this you were referring to?

Comment (DanilYachmenev) [reply docs/Contributor_Scorecard.md:3]: yep

Comment (DanilYachmenev) [reply docs/Contributor_Scorecard.md:3]: but you have to run the `lint_md` script in order for it to be created
when you're done, change PR labels and re-request review

File: docs/Contributor_Scorecard.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+
+## General
+- We give feedback about every 2 weeks.
```
Comment (DanilYachmenev) [inline docs/Contributor_Scorecard.md:6]: lets add an empty line between the headers and text
also, remove dots at the end of the message for format consistency

File: docs/Contributor_Scorecard.md (hunk 11 above)
Comment (sonaalKant) [reply docs/Contributor_Scorecard.md:3]: ohh, okay. will do that.

**Review by samarth9008 — APPROVED** [review]
LG

**Review by DanilYachmenev — COMMENTED** [review]
lgtm

**Pull request merged**